infrastructure/Dockerfile

```diff
@@ -15,8 +15,31 @@ COPY target/${CORFU_TOOLS_JAR} /usr/share/corfu/lib/${CORFU_TOOLS_JAR}
 COPY target/bin /usr/share/corfu/bin
 COPY target/corfu_scripts /usr/share/corfu/corfu_scripts
 COPY target/scripts /usr/share/corfu/scripts
+COPY target/logback.prod.xml /usr/share/corfu/conf/logback.prod.xml
 
 # For integration testing purposes
 COPY target/${CORFU_JAR} /app/corfu.jar
 
-CMD java -cp *.jar org.corfudb.infrastructure.CorfuServer
+ENV JAVA_OPTS -verbose:gc \
+    -XX:+PrintGCDetails \
+    -XX:+PrintGCTimeStamps \
+    -XX:+PrintGCDateStamps \
+    -Xloggc:/var/log/corfu/corfu.jvm.gc.9000.log \
+    -XX:+UseG1GC \
+    -XX:+UseStringDeduplication \
+    -XX:+UseGCLogFileRotation \
+    -XX:NumberOfGCLogFiles=10 \
+    -XX:GCLogFileSize=5M \
+    -XX:+HeapDumpOnOutOfMemoryError \
+    -XX:HeapDumpPath=/var/log/corfu/corfu_oom.hprof \
+    -XX:+HeapDumpOnOutOfMemoryError \
+    -Djdk.nio.maxCachedBufferSize=1048576 \
+    -Dio.netty.recycler.maxCapacityPerThread=0 \
+    -XX:+PrintGCApplicationStoppedTime \
+    -XX:+PrintGCApplicationConcurrentTime \
+    -Djava.io.tmpdir=/image/corfu-server/temp
```

Comment:
`-d DEBUG -k 5000000 --log-size-quota-percentage=40 --metrics -l /config -q eth0 -c 0.3`
Basically can we ensure all the current parameters that corfu server is started with are retained as much as possible?

Reply:
In the docker container, I can only define JAVA_OPTS, everything is defined in helm charts: https://github.com/CorfuDB/corfudb-cloud/blob/c6ab4066223e6bee7a0ffd8852ffbb7fac21342e/cloud/corfu/corfu/templates/statefulset.yaml#L37

Reply:
but what happens if the corfu docker container is started without helm ?